src/inc/clrhost.h

```diff
@@ -83,7 +83,7 @@ extern int RFS_HashStack();
 void ClrFlsAssociateCallback(DWORD slot, PTLS_CALLBACK_FUNCTION callback);
 
 // Function pointer for fast TLS fetch - do not use directly
-typedef LPVOID (*POPTIMIZEDTLSGETTER)();
```

Comment:
note that this and the next fix match a late-bound void function pointer to the proper type (pointer to pointer)